hera_qm/xrfi.py

```diff
@@ -222,9 +251,52 @@ def detrend_medfilt(d, Kt=8, Kf=8):
     return f[Kt:-Kt, Kf:-Kf]
 
 
+def detrend_meanfilt(d, flags=None, Kt=8, Kf=8):
+    """Detrend array using a mean filter.
+    Args:
+        d (array): 2D data array to detrend.
+        flags (array, optional): 2D flag array to be interpretted as mask for d.
+        Kt (int, optional): box size in time (first) dimension to apply medfilt
+            over. Default is 8 pixels.
+        Kf (int, optional): box size in frequency (second) dimension to apply medfilt
+            over. Default is 8 pixels.
+    Returns:
+        f: array of outlier significance metric. Same type and size as d.
+    """
+    # Delay import so astropy is not required for any use of hera_qm
+    from astropy.convolution import convolve
```

Comment:
You may want to mention why you're using astropy rather than scipy's convolve. Maybe just link here: http://docs.astropy.org/en/stable/convolution/

Reply:
Added to comment